tests/e2e/utils/workspace/TestWorkspaceUtil.ts

```diff
@@ -156,14 +156,20 @@ export class TestWorkspaceUtil implements ITestWorkspaceUtil {
             if (stopWorkspaceResponse.status !== 204) {
                 throw new Error(`Can not stop workspace. Code: ${stopWorkspaceResponse.status} Data: ${stopWorkspaceResponse.data}`);
             }
-
+            let stopped: boolean = false;
+            let wsStatus = await this.processRequestHandler.get(stopWorkspaceApiUrl);
             for (let i = 0; i < TestConstants.TS_SELENIUM_PLUGIN_PRECENCE_ATTEMPTS; i++) {
-                const wsStatus = await this.processRequestHandler.get(stopWorkspaceApiUrl);
+                wsStatus = await this.processRequestHandler.get(stopWorkspaceApiUrl);
                 if (wsStatus.data.status === 'STOPPED') {
+                    stopped = true;
                     break;
                 }
                 await this.driverHelper.wait(TestConstants.TS_SELENIUM_DEFAULT_POLLING);
             }
+            if (!stopped) {
+                let waitTime = TestConstants.TS_SELENIUM_PLUGIN_PRECENCE_ATTEMPTS * TestConstants.TS_SELENIUM_DEFAULT_POLLING;
```

Comment:
Actually, `waitTime` doesn't take into account getting status time.
You can use [`process.hrtime()`](https://stackoverflow.com/a/14551263/2556329) to measure spent time more accurately.

Reply:
I don't think we need that precision info, I wanted to print that more like to show what timeouts were set to the user running those tests - so he/she will know if it was waiting for half a minute, two minutes...

Reply:
So, if you don't mind, I'll leave it as it is. But thank you for rising that :)

Reply:
I don't mind to leave it as it.